assets/presto/functions.sdf.yml

```diff
@@ -1389,13 +1423,36 @@ function:
   returns:
     datatype: array<varchar>
 ---
+# VG Adding entries for current_time, current_timestamp, which appear in Trino 449 docs.
+#    DF current_time(), does not carry a time zone, while Trino current_time() does.
+function:
+  name: current_time
+  parameters: []
+  optional-parameters: []
+  returns:
+    datatype: time(p)
+  special: true
+  implemented-by: !datafusion
+---
+# DF now() carries a time zone, but it is hard-wired as UTC currently
+function:
+  name: current_timestamp
+  parameters: []
+  optional-parameters: []
+  returns:
+    datatype: timestamp(p)
+  special: true
+  implemented-by: !datafusion
+    udf: now
+---
```

Comment:
current_time, current_timestamp do not appear in `show functions` but appear in docs
Then we put them in functions_extra.sdf.yml with `special: true`
`special: true` means it can be called without `()`

Reply:
Those actually were there already!  Bu I have added the extra attributes.